device/bgp/bgpsrtesegmentlist.yaml

```diff
@@ -3,16 +3,18 @@ components:
     BgpSrte.SegmentList:
       x-status: under-review
       x-include:
+      - '../../common/common.yaml#/components/schemas/Named.Object'
       - '../device.yaml#/components/schemas/Device.Active'
       description: >-
         Optional container for BGP SR TE Policy segment list settings.
+        The Segment List sub-TLV encodes a single explicit path towards the
+        Endpoint.
       type: object
       properties:
         weight:
           description: >-
-            The weight associated with a given path.
+            The Weight associated with a given path and the sub-TLV is optional.
```

Comment:
The Weight associated with a given path. This sub-TLV ( or is it attribute like in NLRI ? ) is optional .

Reply:
Its a SRTE attribute.